doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,130 @@
+. Enter a *Name* for the provider.
+. From the *Type* list, select *OpenShift Container Platform*.
+. Enter the appropriate *Zone* for the provider. If you do not specify a zone, it is set to `default`.
+. From the *Alerts* list, select *Prometheus* to enable external alerts. Selecting *Prometheus* adds an *Alerts* tab to the lower pane to configure the Prometheus service. Alerts are disabled by default.
+. In the *Default* tab, add the default endpoints for the OpenShift provider: 
+.. Select a *Security Protocol* method to specify how to authenticate the provider:
+* *SSL*: Authenticate the provider securely using a trusted Certificate Authority. Select this option if the provider has a valid SSL certificate and it is signed by a trusted Certificate Authority. No further configuration is required for this option.
+* *SSL trusting custom CA*: Authenticate the provider with a self-signed certificate. For this option, copy your provider’s CA certificate to the *Trusted CA Certificates* box in PEM format.
++
+[NOTE]
+====
+To obtain your OpenShift Container Platform provider's CA certificate, run the `oc get secret` command on your provider, substituting values for your provider and token as needed. To obtain a token for your provider, see link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6-beta/html-single/managing_providers/#Obtaining_OpenShift_Container_Platform_Management_Token[Obtaining an OpenShift Container Platform Management Token].
+
+For example:
+
+  # oc get secret --namespace management-infra management-admin-token-8ixxs --template='{{index .data "ca.crt"}}' | base64 --decode
+  
+Paste the output (a block of text starting with `-----BEGIN CERTIFICATE-----`) into the *Trusted CA Certificates* field.
+====
++
+* *SSL without validation*: Authenticate the provider insecurely (not recommended).
+.. Enter the *Hostname* or IPv4 or IPv6 address of the provider.
++
+[IMPORTANT]
+====
+The *Hostname* must use a unique fully qualified domain name.
+====
++
+.. Enter the *API Port* of the provider. The default port is `8443`.
+.. Click *Validate* to confirm that {product-title_short} can connect to the OpenShift Container Platform provider.
+. Add the Prometheus alerts endpoints in the *Alerts* tab:
+.. Select a *Security Protocol* method to specify how to authenticate the service:
+* *SSL*: Authenticate the provider securely using a trusted Certificate Authority. Select this option if the provider has a valid SSL certificate and it is signed by a trusted Certificate Authority. No further configuration is required for this option.
+* *SSL trusting custom CA*: Authenticate the provider with a self-signed certificate. For this option, copy your provider’s CA certificate to the *Trusted CA Certificates* box in PEM format.
+* *SSL without validation*: Authenticate the provider insecurely using SSL. (Not recommended)
+.. Enter the *Hostname* or IPv4 or IPv6 address of the provider. 
```

Comment:
You have enter here the alert's route from OCP

Reply:
@shalomnaim1 sorry, could you clarify this please?

Reply:
yes,

To configure alerts endpoint you have to use Prometheus alerts route.

you can get this route by running  ``` oc get routes --all-namespaces``` in your OCP master and copy openshift-metrics alerts URL

```{shell}
[root@snaim-ocp-3-7-master ~]# oc get route  --all-namespaces 
NAMESPACE           NAME               HOST/PORT                                                                  PATH      SERVICES           PORT      TERMINATION   WILDCARD
default             docker-registry    docker-registry-default.snaim-ocp-3-7.cfme2.lab.eng.rdu2.redhat.com                  docker-registry    <all>     passthrough   None
default             registry-console   registry-console-default.snaim-ocp-3-7.cfme2.lab.eng.rdu2.redhat.com                 registry-console   <all>     passthrough   None
openshift-infra     hawkular-metrics   hawkular-metrics.snaim-ocp-3-7.cfme2.lab.eng.rdu2.redhat.com                         hawkular-metrics   <all>     reencrypt     None
openshift-metrics   alerts             alerts-openshift-metrics.snaim-ocp-3-7.cfme2.lab.eng.rdu2.redhat.com                 alerts             <all>     reencrypt     None
openshift-metrics   prometheus         prometheus-openshift-metrics.snaim-ocp-3-7.cfme2.lab.eng.rdu2.redhat.com             prometheus         <all>     reencrypt     None
[root@snaim-ocp-3-7-master ~]# 
```
In this case, this would be ```alerts-openshift-metrics.snaim-ocp-3-7....```

Reply:
I would not go into such detail.

If they have the route with the name & namespace we expect the `detect` button would work. Otherwise I would just write enter the `Hostname` (like it is in the UI) or `Route`

Reply:
Understood, thanks.